control/control_performance_analysis/src/control_performance_analysis_node.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 - 2022 Tier IV, Inc., Leo Drive Teknoloji A.Ş.
+// Copyright 2021-2023 UCI SORA Lab, Tier IV, Inc., Leo Drive Teknoloji A.Ş.
```

Comment:
Thank you for your work! But I am not sure whether you can add your company name here or not, because there is not a big change in the code and it does not affect the behavior of the package. In my opinion, you can not add but let's wait for others' answers.
cc @mitsudome-r @xmfcx

Reply:
Essentially, it would be difficult to claim copyright on this modification.
However, I think it is important and motivating for contributors to claim their contributions as the OSS community.

How about preparing the following page as OSS and describing UCI SORA Lab?
https://github.com/all-contributors/all-contributors/blob/master/README.md

Reply:
At the moment we leave it to the decision of the original author of the package.
Even if the author resists to add to the copyright, I think we can consider adding acknowledgement in a different place, likr in readme or in a different file.

Reply:
Mentioning contributions in a different place sounds reasonable to me.